website/pages/docs/migration.mdx

```diff
@@ -349,6 +349,12 @@ export default class Document extends NextDocument {
 
 Here's how to add it for Gatsby:
 
+## Changes
+
+- The Gatsby plugin has been renamed from `gatsby-plugin-chakra-ui` to
+  `@chakra-ui/gatsby-plugin`. Please make sure to have updated this when
+  installing ChakraUI in your next Gatsby project.
```

Comment:
Can you add a space so it's `Chakra UI`?

Reply:
resolved in latest push :)

Reply:
Hmm i don't see a change

Reply:
sorry just came through now, the pushed failed due to my internet haha